libraries/AP_IOMCU/AP_IOMCU.h

```diff
@@ -76,12 +76,12 @@ class AP_IOMCU {
     /*
       get servo rail voltage
      */
-    float get_vservo(void) const { return reg_status.vservo * 0.001; }
+    float get_vservo(void) const { return reg_status.vservo; }
 
     /*
       get rssi voltage
      */
-    float get_vrssi(void) const { return reg_status.vrssi * 0.001; }
+    float get_vrssi(void) const { return reg_status.vrssi; }
```

Comment:
we should probably rename these how you don't actually get a voltage.

Reply:
Fixed